src/ripple/server/impl/ServerHandlerImp.cpp

```diff
@@ -521,7 +520,7 @@ struct ParsedPort
 
     boost::optional<boost::asio::ip::address> ip;
     boost::optional<std::uint16_t> port;
-    boost::optional<bool> allow_admin;
+    boost::optional<std::vector<beast::IP::Address>> admin_ip;
```

Comment:
This vector will not be allocated if no admin setting is specified. Since the setting is optional and the variable was an optional variable before, I kept it as such.